masterlist.yaml

```diff
@@ -2587,6 +2587,38 @@ plugins:
       - crc: 0xAED90543
         util: 'FNVEdit v4.0.4b'
 
+###### Quests and Adventures ######
+  - name: 'NewVegasBounties.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/37310/'
+        name: 'New Vegas Bounties I'
+    req: [ 'SomeguySeries.esm' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ 'NewVegasBounties.esm' ]
+        condition: 'active("NewVegasBounties.esm")'
+      - <<: *useInstead
+        subs: [ '[New Vegas Bounties 1 LE](https://www.nexusmods.com/newvegas/mods/77108/)' ]
+
+  - name: 'NewVegasBounties(\.esm|TTW\.esp)'
```

Comment:
Use this regex instead: `NewVegasBounties(TTW)?\.es[mp]`

Reply:
I can't use this regexp, the original new vegas Bounties would also be included in it. It's why I wrote the regexp this way